config/initializers/devise.rb

```diff
@@ -195,7 +195,7 @@
   # Time interval you can reset your password with a reset password key.
   # Don't put a too small interval or your users won't have the time to
   # change their passwords.
-  config.reset_password_within = 6.hours
+  config.reset_password_within = 48.hours
```

Comment:
This doesn't actually do anything right now. We'll change that later in [AO3-5576](https://otwarchive.atlassian.net/browse/AO3-5576).